web.js

```diff
@@ -24,6 +26,20 @@ var renderIndex = function (req, res) {
   res.render('index', pageData);
 };
 
+var sendMail = function(req, res){
+    var data = req.body;
+
+    transporter.sendMail({
+        from: data.email,
```

Comment:
I think this should be one of our email addresses.  The body can contain the person's email address.